src/libraries/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -433,6 +435,7 @@ public static void X509Certificate2WithT61String()
                 Assert.Equal(certSubject, cert.Issuer);
 
                 Assert.Equal("9E7A5CCC9F951A8700", cert.GetSerialNumber().ByteArrayToHex());
+                Assert.Equal("00871A959FCC5C7A9E", cert.SerialNumberBytes.ByteArrayToHex());
```

Comment:
Can we add a test/assert for:

1. It's returning some cached memory object. `Assert.True(cert.SerialNumberBytes.Span == c1.SerialNumberBytes.Span, "Same SerialNumberBytes memory")`
2. Load the cert, grab `cert.SerialNumberBytes`, dispose the cert, verify the contents of captured serial number.